lib/covertrace/runner/rspec.rb

```diff
@@ -0,0 +1,30 @@
+require "covertrace"
+
+module Covertrace
+  module Runner
+    class RSpec < AbstractRunner
+      def run
+        ARGV.clear
+        ARGV.push(".", *tests_to_run.flat_map { |test| ["--example", test] })
+
+        load Gem.bin_path("rspec-core", "rspec")
+      end
+
+      private
+
+      attr_reader(
+        :dependencies,
+        :merge_base,
+        :root,
+      )
+
+      def tests_to_run
+        changes = Covertrace::GitChanges.changes(merge_base: "origin/master", root: ".")
```

Comment:
does that mean it only works if the current working directory is at the root of the git repo?